src/snowflake/connector/network.py

```diff
@@ -678,6 +681,7 @@ def _get_request(
         url: str,
         headers: dict[str, str],
         token: str = None,
+        params: dict | None = None,
```

Comment:
similar comment as the public api one, although this is a private method and we can update every caller, I personally prefer adding as keyword  argument.